go.mod

```diff
@@ -200,6 +205,11 @@ require (
 	k8s.io/klog/v2 v2.120.1
 )
 
+require (
+	github.com/fsnotify/fsnotify v1.7.0
+	github.com/go-playground/validator/v10 v10.20.0
+)
```

Comment:
nit: Merge with the block above.

Reply:
Non-blocking comment. But already done so will be added in next commit.